contentcuration/contentcuration/management/commands/exportchannel.py

```diff
@@ -299,6 +298,15 @@ def convert_channel_thumbnail(thumbnail):
         encoding = base64.b64encode(file_obj.read()).decode('utf-8')
     return "data:image/png;base64," + encoding
 
+def map_tags_to_node(kolibrinode, ccnode):
+    tags_to_add = []
+
+    for tag in ccnode.tags.all():
+        tags_to_add.append(kolibrimodels.ContentTag.objects.get(pk=tag.pk))
+
+    kolibrinode.tags = tags_to_add
```

Comment:
Hmm this function can be made more succinct, but after writing it out it's harder to read. Let's use this simpler version :)